nlp/front/shared/src/main/kotlin/codec/SentencesDump.kt

```diff
@@ -32,6 +33,7 @@ data class SentencesDump(
 data class SentenceDump(
     val text: String,
     val intent: String,
+    val creationDate : Instant,
```

Comment:
please add `val creationDate : Instant = now().truncatedTo(ChronoUnit.MILLIS)` because deserialization of old dumps will fail

Reply:
Is this actually ever deserialized directly from the database? I was under the impression that this class was only instantiatied by the `ApplicationCodecService`.

Reply:
This is deserialized from the uploaded json file I think :)

Reply:
@Fabilin  any news ?